.github/workflows/labeler.yml

```diff
@@ -4,13 +4,43 @@ on:
     types: [opened]
   pull_request_target:
     types: [opened]
-
+env:
+  MY_GITHUB_TOKEN: ${{ secrets.APM_TECH_USER_TOKEN }}
 jobs:
   triage:
     runs-on: ubuntu-latest
     steps:
-    - uses: AlexanderWert/issue-labeler@v2.3
+    - name: Add agent-nodejs label
+      uses: AlexanderWert/issue-labeler@v2.3
       with:
         repo-token: "${{ secrets.GITHUB_TOKEN }}"
         configuration-path: .github/labeler-config.yml
         enable-versioned-regex: 0
+    - name: Check team membership for user
+      uses: elastic/get-user-teams-membership@v1.0.4
+      id: checkUserMember
+      with:
+        username: ${{ github.actor }}
+        team: 'apm'
+        usernamesToExclude: |
+          apmmachine
+          dependabot
+        GITHUB_TOKEN: ${{ secrets.APM_TECH_USER_TOKEN }}
+    - name: Show team membership
+      run: |
+        echo "::debug::isTeamMember: ${{ steps.checkUserMember.outputs.isTeamMember }}"
+        echo "::debug::isExcluded: ${{ steps.checkUserMember.outputs.isExcluded }}"
```

Comment:
Is this a debugging step that should be removed or commented out before merging?

Reply:
I'd prefer to keep this as this makes understanding simpler when the action fails for some edge cases or bots